frontend/test/metabase/lib/dom.unit.spec.js

```diff
@@ -54,3 +57,41 @@ describe("parseDataUri", () => {
     expect(duration).toBeLessThan(1000);
   });
 });
+
+describe("shouldOpenInBlankWindow", () => {
+  afterEach(() => {
+    EMBEDDING_SDK_CONFIG.isEmbeddingSdk = false;
```

Comment:
FYI, I usually use `mockIsEmbeddingSdk()` which uses `jest.spyOn`, and I assume that gets cleared after tests:

https://github.com/metabase/metabase/blob/3d5410d1996074e739c32f3a6b7386e49aa98b41/frontend/src/metabase/embedding-sdk/mocks/config-mock.ts#L1-L8

Reply:
Thanks! fixed in https://github.com/metabase/metabase/pull/64623/commits/7f51b668165a75f44ab569fe00ecdb8541cca73c